Makefile

```diff
@@ -12,6 +10,7 @@ export AS_FOR_TARGET = llvm-mc$(LLVM_POSTFIX) -arch=aarch64 -mattr=+neon
 export LD_FOR_TARGET = ld.lld$(LLVM_POSTFIX)
 export RANLIB_FOR_TARGET = llvm-ranlib$(LLVM_POSTFIX)
 export CC_FOR_TARGET = clang$(LLVM_POSTFIX) -g -fPIC -ffreestanding -fexceptions -target aarch64-none-linux-gnu -O0 -mtune=cortex-a53 -ccc-gcc-name aarch64-switch-gcc -Wno-unused-command-line-argument -isystem $(realpath $(LIBTRANSISTOR_HOME))/include/
+export CC_FLAGS_FOR_TARGET = $(CC_FLAGS)
```

Comment:
Why are we exporting this? We don't use those `CC_FLAGS` for compiling newlib and don't want to use them for compiling newlib either.

Reply:
Ah, I see. We use them for libssp. No need to change this at the moment then, but I would like to change this the next time I clean up the Makefile.